# HG changeset patch
# User Mike Miller <mtmiller@ieee.org>
# Date 1389755592 18000
#      Tue Jan 14 22:13:12 2014 -0500
# Branch stable
# Node ID de72c443ed3fd6abb22fa1b5236835d96da62f99
# Parent  8671f39d17286ec20ef8a9c135911e785ea701a8
HACKING: Refer to install.txi for the full dependency list (bug #41233)

* HACKING: Refer to install.txi for the complete list of required and
optional dependencies for building Octave.

diff --git a/etc/HACKING b/etc/HACKING
--- a/etc/HACKING
+++ b/etc/HACKING
@@ -49,16 +49,20 @@ the maintainer tools we depend upon, inc
   - gperf <http://www.gnu.org/software/gperf/>
   - Gzip <http://www.gnu.org/software/gzip/>
   - Libtool <http://www.gnu.org/software/libtool/>
   - Mercurial <http://mercurial.selenic.com/>
   - Perl <http://www.cpan.org/>
   - Rsync <http://samba.anu.edu.au/rsync/>
   - Tar <http://www.gnu.org/software/tar/>
 
+In addition to these maintainer tools, Octave makes use of many external
+libraries and packages.  See the file doc/interpreter/install.txi for the
+complete list of required and optional dependencies.
+
 Only building the initial full source tree will be a bit painful.
 Later, after synchronizing from the repository, a plain `make' should
 be sufficient.
 
 ** First clone
 
 If you are reading these notes, you may have already managed to clone
 this package from the repository.  For the record, you will find all the
